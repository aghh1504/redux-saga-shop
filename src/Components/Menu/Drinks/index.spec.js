import React from 'react';
import ReactDOM from 'react-dom';
import Drink from './index';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

describe('Component: App', () => {
  const drinks = [
    {
      "id": 100,
      "name": "Coca",
      "price": 2
  },
  {
      "id": 101,
      "name": "Coca Light",
      "price": 2
  },
  {
      "id": 102,
      "name": "Coca Zéro",
      "price": 2
  },
  {
      "id": 103,
      "name": "Orangina",
      "price": 2
  },
  {
      "id": 104,
      "name": "Shweppes",
      "price": 2
  },
  {
      "id": 105,
      "name": "Seven up",
      "price": 2
  },
  {
      "id": 106,
      "name": "Ice Tea",
      "price": 2
  },
  {
      "id": 107,
      "name": "Perrier 33cl",
      "price": 2
  },
  {
      "id": 108,
      "name": "Badoit 33cl",
      "price": 2
  },
  {
      "id": 109,
      "name": "Jus de Tomate",
      "price": 2
  },
  {
      "id": 110,
      "name": "Jus Pago",
      "price": 2,
      "flavor": [
          "Orange",
          "Mangue",
          "Abricot",
          "Fraise"
      ]
  },
  {
      "id": 111,
      "name": "Kingfisher bière indienne",
      "price": 5
  },
  {
      "id": 112,
      "name": "Duvel",
      "price": 5
  },
  {
      "id": 113,
      "name": "Leffre Brune",
      "price": 5
  },
  {
      "id": 114,
      "name": "Pelforth brune",
      "price": 5
  }
]

  it('should match its empty snapshot', () => {
    const tree = renderer.create(
      <Drink addToBasket={jest.fn()} drinks={drinks}/>
     ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
