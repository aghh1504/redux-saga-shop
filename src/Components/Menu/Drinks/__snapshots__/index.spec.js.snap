// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should match its empty snapshot 1`] = `
<div>
  <div>
    <h4
      className="drinks_name"
    >
      Coca
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={100}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Coca Light
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={101}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Coca Zéro
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={102}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Orangina
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={103}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Shweppes
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={104}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Seven up
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={105}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Ice Tea
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={106}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Perrier 33cl
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={107}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Badoit 33cl
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={108}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Jus de Tomate
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={109}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Jus Pago
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      2
    </h5>
    <button
      className="drinks_checkbox"
      id={110}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Kingfisher bière indienne
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      5
    </h5>
    <button
      className="drinks_checkbox"
      id={111}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Duvel
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      5
    </h5>
    <button
      className="drinks_checkbox"
      id={112}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Leffre Brune
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      5
    </h5>
    <button
      className="drinks_checkbox"
      id={113}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
  <div>
    <h4
      className="drinks_name"
    >
      Pelforth brune
    </h4>
    <h5
      className="drinks_price"
    >
      Price $
      5
    </h5>
    <button
      className="drinks_checkbox"
      id={114}
      onClick={[Function]}
    >
      Add to basket
    </button>
  </div>
</div>
`;
